lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -1510,7 +1510,7 @@ def default_host_ip(self):
                         break
                 except NotFound as dummy:
```

Comment:
Rename this to `e`, and use `e` in the message.  Instead of `dummy` which is mean to be a throw away var that is not used.